pkg/operator/ceph/csi/template/multus-host-setup.yaml

```diff
@@ -0,0 +1,47 @@
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: csi-multus
+  namespace: {{ .Namespace }}
+spec:
+  selector:
+    matchLabels:
+      name: rook-ceph-multus
+  updateStrategy:
+    type: OnDelete
+  template:
+    metadata:
+      labels:
+        name: rook-ceph-multus
+        app: rook-ceph-multus
+      annotations:
+        k8s.v1.cni.cncf.io/networks: {{ .MultusNetwork }}
+    spec:
+      serviceAccount: rook-ceph-multus
+      priorityClassName: system-node-critical
+      initContainers:
+      - name: if-migration
+        image: {{ .MultusImage }}
+        command: ['sh', '-c', 'ip --json netns list-id | jq -j ".[] | select(.nsid == 0).name" > /tmp/hostns.txt && if-mover']
```

Comment:
We cannot assume the `ip` command is available, especially downstream it's not. Also on my machine this command returns nothing since the key `name` is not part of the output.

Also, I'm wondering if parsing `/var/run/netns`wouldn't be sufficient or if `if-mover` could detect it from its lib?

Reply:
Also if we still need `/tmp/hostns.txt` we need to add a volume for this in the container spec.